streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -250,7 +250,7 @@ class ZStream[-R, +E, +A](val process: ZManaged[R, E, Pull[R, E, A]]) extends Se
    * Aggregated elements will be fed into the schedule to determine the delays between
    * pulls.
    */
-  final def aggregateWithin[R1 <: R, E1 >: E, A1 >: A, B, C](
+  final def aggregateWithinEither[R1 <: R, E1 >: E, A1 >: A, B, C](
```

Comment:
We will need to add some more documentation to this method, to highlight the difference compared to the new `aggregateWithin` which currently doesn't have a scaladoc.

Reply:
I appreciate your feedback! I've added ScalaDoc for both methods in https://github.com/zio/zio/pull/1700/commits/a4b8851cafcfc94722a5f88180fec3d56bad1abe.

Reply:
@iravid Should we open an issue maybe for the hackathon, to add annotated documentation like this?

Reply:
See https://github.com/zio/zio/issues/413 where I started adding ScalaDoc to some files. It's a good first issue for a hackathon.